src/loader.rs

```diff
@@ -48,6 +48,30 @@ pub struct LoaderMetadata {
     encrypted_bytes: CipherText,
 }
 
+// Loader for using in tests.  Just creates defualt metadata and Keytree.
+// This shouldn't be used for production, only tests.
```

Comment:
This should be guarded with `#[cfg(any(test, feature = "testing"))]`, and maybe named something more obviously test-related, e.g. `TestKeystoreLoader`.